.gitignore

```diff
@@ -253,6 +253,7 @@ node_modules/
 /ajax/libs/numbered/1.0.0
 /ajax/libs/jquery-bracket/release-*
 /ajax/libs/jquery.ime/1.0.0+20131229
+/ajax/libs/typebase.css/v0.5
```

Comment:
Should be `/ajax/libs/typebase.css/0.5`

Reply:
Are you sure? The tag on lib is v0.5
![screen shot 2018-05-21 at 13 52 20](https://user-images.githubusercontent.com/11975561/40304169-46d0b606-5cfe-11e8-8865-1e619cb81af1.png)

Reply:
@sashberd Pretty sure. The one way to check this to use the command `git status` to see if you can track v0.5 using Git.
When you used `/ajax/libs/typebase.css/v0.5`, the local folder `/ajax/libs/typebase.css/0.5` and its content were still tracked by Git.